README.md

```diff
@@ -1 +1,140 @@
+     ```sh
+    sudo usermod -a -G kvm $(whoami)
+    sudo usermod -a -G libvirt $(whoami)
+    ```
+    3.a. Now you can check user permissions by command:
+    ```sh
+    id
+    ```
+    3.b. Or you can check user persmissions manually:
+    ```sh
+    cat /etc/group | grep kvm
+    cat /etc/group | grep libvirt
+    ```
+   >_Remember: If user has `0` it means that user is `root`, numbers `1000+` means that's normal user_
+4. Get admin permissions:
+    ```sh
+    sudo -i
+    ```
+5. Start libvirtd and check status:
+    ```sh
+    systemctl start libvirtd
+    systemctl status libvirtd
+    ```
+6. Download system image from [Ubuntu Cloud Image][UbCi], and copy image from `/Downloads` to libwirt directory, for example:
+    ```sh
+    cd ~/Downloads/
+    sudo cp bionic-server-cloudimg-amd64.img /var/lib/libvirt/images/
+    ```
+    6.a (Optional) You can make copy of image to use it in the future
+    ```sh
+    sudo cp /var/lib/libvirt/images/bionic-server-cloudimg-amd64.img /var/lib/libvirt/images/vm1.qcow2
+    ```
+    >Note, that you can change name of your image, by changing `vm1`_
+    
+7. Set password for root user, using previously installed `libguestfs`:
+    ```sh
+    sudo virt-customize -a /var/lib/libvirt/images/vm1.qcow2 --root-password password:<YOUR_PASSWORD>
+    ```
+    _Change `<YOUR_PASSWORD>`, as you want_
+8. Check ip adress of your Virtual Machine:
+    ```sh
+    ip a
+    ```
+    8.a If there is no ip adress, you may start dhclient service:
+    ```sh
+    dhclient
+    ```
+9. Configure ssh access:
+    ```sh
+    dpkg-reconfigure openssh-server
+    systemctl restart ssh
+    ```
+10. Install Terraform framework:
+    ```sh
+    sudo -i
+    curl -fsSL https://apt.releases.hashicorp.com/gpg | sudo apt-key add -
+    sudo apt-add-repository "deb [arch=$(dpkg --print-architecture)] https://apt.releases.hashicorp.com $(lsb_release -cs) main"
+    apt install terraform
+    ```
+11. Start Terraform:
+    ```sh
+    terraform init
+    terraform plan
+    terraform apply -auto-approve
+    ```
+12. After all, you can clean up using:
+    ```sh
+    terraform destroy -auto-approve
+    ```
+    
+## Technologies used
+
+Cloud-infra uses a number of open source projects to work properly:
+
+- [Terraform][TF] - codifies cloud APIs into declarative configuration files
+- [Kernel Virtual Machine][KVM] - is a full virtualization solution for Linux on x86 hardware containing virtualization extensions
+- [Libvirt][LV] - is a toolkit to manage virtualization platforms
+- [Bridge Utils][BU]- a utility needed to create, manage and setting up networks for a hosted virtual machine
+- [Virtual Machine Manager][VMM]-  is a desktop user interface for managing virtual machines through libvirt
+- [Git][GIT]- version control system
+- [Virtual Machine Manager][LGFS]-  tools for accessing and modifying virtual machine disk images
+
+And of course Cloud-infra is open source with a [repository][repo] on GitHub.
+
+## Plugins
```

Comment:
I would change it to `Requirements` instead

Physical server or PC with:
8x CPU cores
24GB of Ram
256GB free space on hard disk
Linux OS installed